<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.875818, y1:0.585, x2:0.511, y2:0.584955, stop:0.210227 rgba(103, 217, 103, 244), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="login_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>801</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 42pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;
background-color: qlineargradient(spread:reflect, x1:0.875818, y1:0.585, x2:0.488273, y2:0.591, stop:0.210227 rgba(103, 217, 103, 244), stop:0.977273 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_field">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_field">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Times New Roman&quot;;
background-color: qlineargradient(spread:reflect, x1:0.875818, y1:0.585, x2:0.488273, y2:0.591, stop:0.210227 rgba(103, 217, 103, 244), stop:0.977273 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 10pt &quot;Times New Roman&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
